openshift-ansible.spec

```diff
@@ -78,6 +79,8 @@ popd
 %files
 %doc LICENSE.md README*
 %dir %{_datadir}/ansible/%{name}
+%{_datadir}/ansible/%{name}/library
+%ghost %%{_datadir}/ansible/%{name}/playbooks/common/openshift-master/library.rpmmoved
```

Comment:
typo - one too many % before datadir, should be  
+%ghost %{_datadir}/ansible/%{name}/playbooks/common/openshift-master/library.rpmmoved